### PR TITLE
FOPTS-4039 Bug fix for unitofMeasure for ultra sized disk

### PR DESCRIPTION
**Pull request by Kaaku2302**
Updated /corrected for Ultra pricing

### Description
There is a bug in calculation of the Ultra Price as logic errors out at the unitOfMeasure calculation. This PR fixes the bug

### Issues Resolved
Issue with unitOfMeasure calculation and errors indicator by the static checks

### Link to Example Applied Policy
Tested in the customer landscape

### Contribution Check List

- [ ] New functionality includes testing.
- [ ] New functionality has been documented in the README if applicable
- [ ] New functionality has been documented in CHANGELOG.MD

### CODE REVIEW

**Review by nia-vf1 — CHANGES_REQUESTED** [review]

File: cost/azure/rightsize_managed_disks/CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## v2.0.1
+
+- Fixed error related to the calculation of unitOfMeasure for the method findUltraPrice for ultra sized disk
```
Comment (midir99) [inline cost/azure/rightsize_managed_disks/CHANGELOG.md:5]: - Fixed bug that caused wrong calculations for ultra tier disks

Comment (Kaaku2302) [reply cost/azure/rightsize_managed_disks/CHANGELOG.md]: Done

File: cost/azure/rightsize_managed_disks/azure_rightsize_managed_disks.pt [hunk 2]
```diff
@@ -495,10 +495,10 @@ script "js_azure_virtual_machines_with_metrics", type: "javascript" do
     path: vmId + "/providers/microsoft.insights/metrics",
     query_params: {
       "api-version": "2018-01-01",
-      "metricnames": "OS Disk IOPS Consumed Percentage,OS Disk Bandwidth Consumed Percentage,Data Disk IOPS Consumed Percentage,Data Disk Bandwidth Consumed Percentage",
+      "metricnames": "OS Disk IOPS Consumed Percentage, OS Disk Bandwidth Consumed Percentage, Data Disk IOPS Consumed Percentage, Data Disk Bandwidth Consumed Percentage",
```
[the remark this answers is not in the document]
Comment (Kaaku2302) [reply cost/azure/rightsize_managed_disks/azure_rightsize_managed_disks.pt]: Fixed

**Comment by nia-vf1** [comment]
@midir99 are you happy for this to be merged, or are there any outstanding changes requested?

**Comment by midir99** [comment]
@nia-vf1 I'm happy with it being merged :)

**Review by midir99 — APPROVED** [review]

**Pull request merged**